app/lib/efile/ny/it201.rb

```diff
@@ -3,63 +3,70 @@ module Ny
     class It201
       attr_reader :lines
 
-      def initialize(year:, filing_status:, claimed_as_dependent:, dependent_count:, input_lines:, it227:)
+      def initialize(year:, filing_status:, claimed_as_dependent:, dependent_count:, input_lines:, it213:, it214:, it215:, it227:)
         @year = year
 
         @filing_status = filing_status # single, married_filing_jointly, that's all we support for now
         @claimed_as_dependent = claimed_as_dependent # true/false
         @dependent_count = dependent_count # number
+
+        @value_access_tracker = ValueAccessTracker.new
```

Comment:
I think I'd prefer if the caller is responsible for setting up the `ValueAccessTracker`. However, I haven't fully taken the time to understand the new `set_line` API yet.

Reply:
I restored `.value_access_tracker=`